src/pages/github-teacher-toolbox.tsx

```diff
@@ -58,7 +58,7 @@ const offers: PricingBoxProps[] = [
 ]
 
 const GithubTeacherToolBoxPage: React.SFC<{}> = () => (
-    <IndexLayout canonical="/github-teacher-toolbox/" title="GitHub Teacher Toolbox">
+    <IndexLayout canonical="/github-teacher-toolbox/" title="GitHub Teacher Toolbox" description="With Gitpod you can create exams and exercises easily, help out your students via live tutoring and benefit from a simple onbaoarding from any device.">
```

Comment:
Adjusted description. Please also reflect this change inside the page (paragraph after the headings).

```suggestion
    <IndexLayout canonical="/github-teacher-toolbox/" title="GitHub Teacher Toolbox" description="With Gitpod you can create coding exams and exercises easily, help out your students via live tutoring, and benefit from a simple onboarding with any device.">
```